core/services/bulletprooftxmanager/eth_broadcaster_test.go

```diff
@@ -59,7 +60,11 @@ func TestEthBroadcaster_ProcessUnstartedEthTxs_Success(t *testing.T) {
 	ethClient := new(mocks.Client)
 	store.EthClient = ethClient
 
-	eb := bulletprooftxmanager.NewEthBroadcaster(store, config)
+	eventBroadcaster := postgres.NewEventBroadcaster(config.DatabaseURL(), 0, 0)
+	eventBroadcaster.Start()
+	defer eventBroadcaster.Stop()
+
+	eb := bulletprooftxmanager.NewEthBroadcaster(store, config, eventBroadcaster)
```

Comment:
```suggestion
	eb, cleanup := cltest.NewEthBroadcaster(store)
	defer cleanup()
```
Worth making a cltest factory for?